v2/apiserver/internal/authx/common_test.go

```diff
@@ -0,0 +1,44 @@
+package authx
+
+import (
+	"encoding/json"
+	"reflect"
+	"testing"
+	"unsafe"
+
+	"github.com/brigadecore/brigade/v2/apiserver/internal/meta"
```

Comment:
@radu-matei I am missing something here, possibly...

Is there something wrong with the import order that neither I or the linter are spotting?

Reply:
Going to not let this block the PR since @technosophos has LGTM'ed. But please do open an issue if there is something outstanding here that I'm not spotting.

Reply:
I should have been more explicit - IIRC, the previous linter settings for import order were enforcing something like:

https://github.com/brigadecore/brigade/blob/66dfceb3de22d5a735615d82d31077bf6a512a12/pkg/storage/kube/worker.go#L3-L15

Reply:
Oh... new linter settings got copied over from the prototype. I'm not opposed to what you have above. It makes sense to me. Do you know if gofmt or goimports can be configured somehow to format imports that way automatically?